modules/core_project_factory/versions.tf

```diff
@@ -20,11 +20,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.50, < 6"
+      version = ">= 3.64, < 6"
```

Comment:
not marking as breaking since this is Internal module and modules using this module are already 4+